### PR TITLE
Write cli output to file

### PR DESCRIPTION
**Pull request by awcrosby**

### CODE REVIEW

File: galaxy_importer/main.py [hunk 1]
```diff
@@ -16,37 +16,62 @@
...
 
-    :param filepath: Artifact file to import.
-    """
-    json_data = collection.import_collection(filepath)
-    # TODO: make default output go to file
-    print(json_data)
+    json_data = call_importer(filepath=args.filepath)
+
+    if args.print_result:
+        pprint(json.loads(json_data))
```
Comment (cutwater) [inline galaxy_importer/main.py]: Is this for debug purposes or supposed to be used by parent program?

In any case using `pprint` for JSON formatting is a terrible idea. It is not guaranteed that `pprint` will provide a valid formatted JSON output. You should use `json.dump() / json.dumps()` with `indent=` parameter for prettifying JSON.

File: galaxy_importer/main.py [hunk 2]
```diff
@@ -16,37 +16,62 @@
...
+
+    write_output_file(json_data)
 
 
 def parse_args(args):
     parser = argparse.ArgumentParser(
         description='Import collection and run linting')
     parser.add_argument(
-        '-f', '--filepath', default=None, help='Artifact to import.')
+        '-f', '--filepath',
```
Comment (cutwater) [inline galaxy_importer/main.py]: Minor \ Naming: IMO `filepath` is too long, usually `file` or `input` or `artifact` is enough.

Also, if artifact file is a required argument for importer, usually it's better to make it a positional argument.

File: galaxy_importer/main.py [hunk 3]
```diff
@@ -16,37 +16,62 @@
...
+        default=None,
+        required=True,
+        help='Artifact to import.')
+    parser.add_argument(
```
Comment (cutwater) [inline galaxy_importer/main.py]: What is the behavior and use case for this parameter, when it's set to `True`?

Comment (awcrosby) [reply galaxy_importer/main.py]: For debug purposes to see full result of importer printed to the console

File: galaxy_importer/main.py (hunk 2 above)
Comment (awcrosby) [reply galaxy_importer/main.py]: Updated

File: galaxy_importer/main.py (hunk 1 above)
Comment (awcrosby) [reply galaxy_importer/main.py]: This is just for debugging, updated.

@cutwater do you think we should remove the `json.dumps()` from what the galaxy_importer returns and let the caller do that?: https://github.com/ansible/galaxy-importer/blob/master/galaxy_importer/collection.py#L47

Comment (cutwater) [reply galaxy_importer/main.py]: @awcrosby yes, I think it's a better option, then you can decide on formatting when writing data to the output.

File: galaxy_importer/main.py (hunk 3 above)
Comment (cutwater) [reply galaxy_importer/main.py]: Ok, thanks. 

Minor: usually in CLI args we use dashes as word separators (e.g. `--print-result`)

**Pull request merged**